atomic_reactor/__init__.py

```diff
@@ -29,7 +37,10 @@ def set_logging(name="atomic_reactor", level=logging.DEBUG, handler=None):
         handler.setLevel(logging.DEBUG)
 
         # create formatter
-        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        # NOTE: "platform:%(arch)s" is expected to be parsed, update parser in
+        # NOTE: osbs-client if this needs updating
+        fmt = '%(asctime)s platform:%(arch)s - %(name)s - %(levelname)s - %(message)s'
```

Comment:
Since it depends on osbs-client implementation it might be a good idea to move it to `atomic_reactor/constants.py`?